x-pack/plugins/apm/public/components/shared/popover_tooltip/index.tsx

```diff
@@ -30,7 +35,7 @@ export function PopoverTooltip({ ariaLabel, children }: PopoverTooltipProps) {
           }}
           size="xs"
           color="primary"
-          iconType="questionInCircle"
+          iconType={iconType ?? 'questionInCircle'}
```

Comment:
Then you can skip the condition here:

```suggestion
          iconType={iconType}
```